clay.h

```diff
@@ -3897,24 +3897,23 @@ void Clay_SetCurrentContext(Clay_Context* context) {
 }
 
 CLAY_WASM_EXPORT("Clay_GetScrollOffset")
-Clay_Vector2 Clay_GetScrollOffset() {
+Clay_Vector2 Clay_GetScrollOffset(void) {
     Clay_Context* context = Clay_GetCurrentContext();
     if (context->booleanWarnings.maxElementsExceeded) {
-        return CLAY__INIT(Clay_Vector2){};
+        return CLAY__INIT(Clay_Vector2){0};
```

Comment:
This should use the `CLAY__DEFAULT_STRUCT` macro, which uses either `{}` or `{0}` depending on whether it's C++ or C.

Reply:
Thank you for letting me know, fixed.